src/main/java/Parser.java

```diff
@@ -0,0 +1,93 @@
+import command.*;
```

Comment:
wildcard import is not encouraged according to Java coding standard. It would be better to list them out explicitly.